api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -647,9 +647,6 @@ public static String getOpenmrsProperty(String property) {
 		props.add(new GlobalProperty("dashboard.overview.showConcepts", "",
 		        "Comma delimited list of concepts ids to show on the patient dashboard overview tab"));
 		
-		props.add(new GlobalProperty(GP_DASHBOARD_CONCEPTS, "5497",
```

Comment:
looking good i want to make sure that the dropped concepts here do also exist in our concept dictionary

Reply:
I am not sure because the ticket specified that the code will be transferred to the starter dictionary if any is setup